lib/jquery.tinycarousel.js

```diff
@@ -239,7 +250,6 @@
                     }
                 });
 
-            _setButtons();
```

Comment:
This is the one change I'm not 100% sure won't have side effects. When the plugin first loads, `_intialize()` calls `update()` which in turn calls `_setButtons()`. `_initialize()` then calls `move()` which had the `_setButtons()` call above. This was causing the `disable` class to get set appropriately on the first call and then turned off on this second call.